organization/_posts/2017-11-29-packaging.md

```diff
@@ -0,0 +1,85 @@
+* Noted second draft of HSF White Paper open.
+  * Encourage people in this WG to read and comment, especially Software Development.
+  * Also for people to sign - don’t have to have contributed to the content to sign.
+* Update on Use Cases Document
+  * Good feedback and comments since last time, thanks to contributors!
+  * Common agreement on "must have" requirements: specify build options for dependencies,
+    reuse of binary artifacts, setup of runtime "views".
+  * Clarification on: global build options, recipe hierarchy, install relocation (with example
+    implementations), development environment.
+  * Many comments are useful in their own right, so need to capture this information
+    somewhere other than a Google Docs comment thread.
+  * Clarifications with Giulio Eulisse on his comments
+    * On sources, felt important to have a self-managed backup of third-party sources.
+      Guarantees future availability, plus allowing patching. Comment from ATLAS
+      that they also install package sources as part of their package/distribution
+      pipeline.
+    * On reducing command line options, this is to keep major things like switching
+      compiler part of the recipe.
+    * On recipes, importance is to ensure that older packages can be rebuilt in the future.
+      E.g. in Alibuild, Alibuild as it is now can build Alibuild recipes from N years ago.
+  * Clear that we need to identify "actors" in the Use Cases, e.g. librarian, end user.
+
+## Presentation from Oana: ATLAS Releases: Build, Packaging and Deployment - short overview
+* ATLAS offline code is split into many, many units of code called “packages” (usually implementing some particular classes and building a small number of libraries). The full offline Athena release has about 2000 packages.
+* New way of building Athena is quite simple. LCG release is a base, then build AtlasExternals, then Gaudi, then the rest of Athena.
+* Build of Athena bootstraps using AtlasSetup  (sets up gcc, cmake version etc). A few helper scripts used for building whole of Athena, or subsets - output is rpm, from CPack.
+* Nightly rpms are release candidates - copied to EOS. Distribution to CVMFS via Python script. Each nightly build put into date-time tagged directories, under this are the Athena, AthenaExternals, Gaudi. LCG, Externals, tdaq are common over a range of Nightlies
+* Also allow private builds to be distributed over CVMFS
+* Future: looking at ways to save AtlasExternals binary artifacts. Reuse? Looking at Nexus, git LSF. Breaking up large rpms. Include build of LCG
+
+### Discussion
+* Microsoft has released source code for their Git vistual filesystem (GVFS). There is a project on GitHub to port this to Mac and then Linux. Will be a talk on this at CVMFS Workshop in January. Background: Windows source code is very large, so using a Virtual File System to only checkout files when touched.
```

Comment:
Not a project "on Github" by a project by Github and Microsoft, at least as far as I understand it. The whole story at https://arstechnica.com/gadgets/2017/11/microsoft-and-github-team-up-to-take-git-virtual-file-system-to-macos-linux/